src/components/Common/UI/Calendar/Calendar.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import type { CalendarProps } from './CalendarTypes'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
```

Comment:
This file was just moved from the original location to the UI directory, it's not net new. Compare to https://github.com/Gusto/embedded-react-sdk/blob/main/src/components/Common/CalendarDisplay/CalendarDisplay.stories.tsx